src/main/java/me/illia/jlog/JLogger.java

```diff
@@ -1,53 +1,104 @@
 package me.illia.jlog;
 
-import java.util.Random;
+import me.illia.jlog.log.LogFile;
+import me.illia.jlog.other.JLogColor;
+import me.illia.jlog.other.JLogTimestamp;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
 
 public class JLogger {
-    public static JLoggerStyle style;
-    // The return type of the log() method represents did the program fail, did it succeed, or did it 'super failed' aka critical error
-    public static int log(JLogLvl lvl, Object msg) {
-        int result = -2;
-
-        if (style == null) {
-            result = -1;
-            throw new IllegalArgumentException("You should call the method 'setStyle' before logging anything to the console!");
-        }
+    private static final Scanner IN = new Scanner(System.in);
+    public static Date date = new Date();
+    public static String LOG_PATH = "*/latest.log";
+    public static ArrayList<String> LOGS = new ArrayList<>();
+    private static JLoggerStyle style = new JLoggerStyle();
 
+    public static void log(JLogLvl lvl, Object msg) {
         String stringMsg = msg.toString();
 
+        if (!stringMsg.endsWith("\r")) stringMsg += '\r';
```

Comment:
here would be the best place to initialize JLog.date

Reply:
ok, i think i did that now. Check the latest commit, like that?